2019-11-10 20:38:28,340 DEBUG   MainThread:12736 [wandb_config.py:_load_defaults():110] no defaults not found in config-defaults.yaml
2019-11-10 20:38:28,383 DEBUG   MainThread:12736 [util.py:is_cygwin_git():318] Failed checking if running in CYGWIN due to: FileNotFoundError(2, 'The system cannot find the file specified', None, 2, None)
2019-11-10 20:38:28,388 DEBUG   MainThread:12736 [cmd.py:execute():724] Popen(['git', 'cat-file', '--batch-check'], cwd=C:\Users\kyuree\dev\ai601, universal_newlines=False, shell=None, istream=<valid stream>)
2019-11-10 20:38:28,458 DEBUG   MainThread:12736 [cmd.py:execute():724] Popen(['git', 'rev-parse', '--show-toplevel'], cwd=C:\Users\kyuree\dev\ai601, universal_newlines=False, shell=None, istream=None)
2019-11-10 20:38:28,526 DEBUG   MainThread:12736 [cmd.py:execute():724] Popen(['git', 'status', '--porcelain', '--untracked-files'], cwd=C:\Users\kyuree\dev\ai601, universal_newlines=False, shell=None, istream=None)
2019-11-10 20:38:28,604 DEBUG   MainThread:12736 [run_manager.py:__init__():535] Initialized sync for gnnproject/bv0vw1nm
2019-11-10 20:38:28,613 INFO    MainThread:12736 [run_manager.py:wrap_existing_process():1131] wrapping existing process 2204
2019-11-10 20:38:28,618 DEBUG   MainThread:12736 [connectionpool.py:_new_conn():959] Starting new HTTPS connection (1): pypi.org:443
2019-11-10 20:38:29,229 DEBUG   MainThread:12736 [connectionpool.py:_make_request():437] https://pypi.org:443 "GET /pypi/wandb/json HTTP/1.1" 200 37669
2019-11-10 20:38:29,367 INFO    Thread-18 :12736 [run_manager.py:_on_file_created():671] file/dir created: C:\Users\kyuree\dev\ai601\wandb\run-20191110_113825-bv0vw1nm\wandb-metadata.json
2019-11-10 20:38:29,558 INFO    MainThread:12736 [run_manager.py:init_run():916] system metrics and metadata threads started
2019-11-10 20:38:29,558 INFO    MainThread:12736 [run_manager.py:init_run():950] upserting run before process can begin, waiting at most 10 seconds
2019-11-10 20:38:29,595 DEBUG   Thread-43 :12736 [connectionpool.py:_new_conn():959] Starting new HTTPS connection (1): api.wandb.ai:443
2019-11-10 20:38:30,073 DEBUG   Thread-43 :12736 [connectionpool.py:_make_request():437] https://api.wandb.ai:443 "POST /graphql HTTP/1.1" 200 484
2019-11-10 20:38:30,110 INFO    Thread-43 :12736 [run_manager.py:_upsert_run():1035] saving patches
2019-11-10 20:38:30,112 DEBUG   Thread-43 :12736 [cmd.py:execute():724] Popen(['git', 'rev-parse', '--show-toplevel'], cwd=C:\Users\kyuree\dev\ai601, universal_newlines=False, shell=None, istream=None)
2019-11-10 20:38:30,197 DEBUG   Thread-43 :12736 [cmd.py:execute():724] Popen(['git', 'diff', '--cached', '--abbrev=40', '--full-index', '--raw'], cwd=C:\Users\kyuree\dev\ai601, universal_newlines=False, shell=None, istream=None)
2019-11-10 20:38:30,260 DEBUG   Thread-43 :12736 [cmd.py:execute():724] Popen(['git', 'diff', '--abbrev=40', '--full-index', '--raw'], cwd=C:\Users\kyuree\dev\ai601, universal_newlines=False, shell=None, istream=None)
2019-11-10 20:38:30,354 DEBUG   Thread-43 :12736 [cmd.py:execute():724] Popen(['git', 'version'], cwd=C:\Users\kyuree\dev\ai601, universal_newlines=False, shell=None, istream=None)
